zebra-rpc/src/server.rs

```diff
@@ -87,6 +72,12 @@ impl RpcServer {
             + 'static,
         State::Future: Send,
         Tip: ChainTip + Clone + Send + Sync + 'static,
+        BlockVerifier: Service<Arc<Block>, Response = block::Hash, Error = zebra_consensus::BoxError>
+            + Clone
+            + Send
+            + Sync
+            + 'static,
+        <BlockVerifier as Service<Arc<Block>>>::Future: Send,
```

Comment:
Using a generic here makes some tests easier to write, because we can use a `MockService`.